website/src/tabs/develop/execute/CodeEditor.jsx

```diff
@@ -0,0 +1,84 @@
+import CodeMirror from "@uiw/react-codemirror";
+import { okaidia } from "@uiw/codemirror-theme-okaidia";
+import { noctisLilac } from "@uiw/codemirror-theme-noctis-lilac";
+import { simpleMode } from "@codemirror/legacy-modes/mode/simple-mode";
+import { StreamLanguage } from "@codemirror/language";
+import { theme } from "antd";
+import { useState } from "react";
+
+const aleoSyntaxHighlight = {
+    start: [
+        {
+            regex: /(?:^|\s)(function|program|as|by|interface|closure|into|import)(?:$|\s)/,
+            token: "keyword",
+        },
+        {
+            regex: /(?:^|\s)(finalize|mapping|increment|decrement)(?:$|\s)/,
```

Comment:
These opcodes are deprecated in aleo instructions.
I have added an equivalent issue on `aleo-docs` to remove them from the syntax highlighter.
```suggestion
            regex: /(?:^|\s)(finalize|mapping)(?:$|\s)/,
```